charts/linkerd-control-plane/values.yaml

```
@@ -192,6 +192,9 @@ proxyInit:
   # -- Default set of outbound ports to skip via iptables
   # - Galera (4567,4568)
   ignoreOutboundPorts: "4567,4568"
+  # -- Default set of ports to skip via iptables for control plane
+  # -- components so they can communicate with the Kubernetes API Server
```

It looks like this makes the comment separate.

```suggestion
  # components so they can communicate with the Kubernetes API Server
```

---

Nice, good spot!